lib/circuit.js

```diff
@@ -171,6 +176,9 @@ class CircuitBreaker extends EventEmitter {
 
     // Register with the hystrix stats listener
     this[HYSTRIX_STATS] = new HystrixStats(this);
+    if (PrometheusMetrics && options.usePrometheus) {
```

Comment:
What happens if `options.usePrometheus` is `true`, but `PrometheusMetrics` is undefined? Obviously, this conditional check will be false, but should we emit an error in that case?

Reply:
I think that part is more related to webpack config like the other situation commented above.
Prometheus is now enabled on two situations:
1) Web vs Node environment (webpack)
2) User defined option ( `usePrometheus` )

But yes, is good to emit an error in case the user try to use prometheus without `usePrometheus` option :+1: